Assets/Scripts/Tank/Enemy/EnemyController.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System.Collections;
+using UnityEngine;
+[RequireComponent(typeof(Animator))]
+public class EnemyController : MonoBehaviour, Idamagable, IAttacker
```

Comment:
Change to IDamagable (C# Naming Conventions)

Reply:
Done.